go-update/src/test/java/org/reactome/release/goupdate/JavaVsPerlComparison.java

```diff
@@ -107,94 +156,127 @@ public int compare(GKInstance o1, GKInstance o2)
+									accession + "' " + "there are " + i + " differences:\n" + sb.toString());
 						}
 						else
 						{
+							sameNameToCount.put(perlInst.getExtendedDisplayName(), i);
 							sameCount.incrementAndGet();
 						}
 						Predicate<? super SchemaAttribute> isNameAttribute = p-> {
-								return p.getName().equals("name") || p.getName().equals("_displayName");
-							};
-						
-						// Now, we need to compare referrers, since they might have display name changes. Want to make sure we capture those correctly.
+							return p.getName().equals("name") || p.getName().equals("_displayName");
+						};
+
+						// Now, we need to compare referrers, since they might have display name changes.
+						// Want to make sure we capture those correctly.
 						if (javaGoInst.getSchemClass().getName().equals(ReactomeJavaConstants.GO_BiologicalProcess))
 						{
 							StringBuilder sb1 = new StringBuilder();
-	
+
 							@SuppressWarnings("unchecked")
-							List<GKInstance> javaList = ((List<GKInstance>) javaGoInst.getReferers(ReactomeJavaConstants.goBiologicalProcess));
+							List<GKInstance> javaList =
+								((List<GKInstance>) javaGoInst.getReferers(ReactomeJavaConstants.goBiologicalProcess));
 							@SuppressWarnings("unchecked")
-							List<GKInstance> perlList = ((List<GKInstance>) perlInst.getReferers(ReactomeJavaConstants.goBiologicalProcess));
+							List<GKInstance> perlList =
+								((List<GKInstance>) perlInst.getReferers(ReactomeJavaConstants.goBiologicalProcess));
 							if (javaList != null && perlList != null)
 							{
 								javaList = javaList.stream().sorted(dbIdComparator).collect(Collectors.toList());
 								perlList = perlList.stream().sorted(dbIdComparator).collect(Collectors.toList());
-	
+
 								GKInstance javaReferrer =  javaList.get(0);
 								GKInstance perlReferrer = perlList.get(0);
-	
-								int numDiffs = DBObjectComparer.compareInstances(javaReferrer, perlReferrer, sb1, 0, isNameAttribute, true);
+
+								int numDiffs = DBObjectComparer.compareInstances(
+									javaReferrer, perlReferrer, sb1, 0, isNameAttribute, true
+								);
 								if (numDiffs > 0)
 								{
 									regulationDiffCount.incrementAndGet();
-									mainSB.append("\n***\nFor Java-updated referring instance \""+javaReferrer.toString()+"\" there are " + numDiffs + " differences:\n"+sb1.toString());
+									mainSB.append(
+										"\n***\n" +
+											"For Java-updated referring instance '" + javaReferrer + "' there are " +
+											numDiffs + " differences:\n" + sb1.toString());
 								}
 								else
 								{
 									regulationSameCount.incrementAndGet();
 								}
 							}
 						}
-						else if (javaGoInst.getSchemClass().getName().equals(ReactomeJavaConstants.GO_CellularComponent))
+						else if (
+							javaGoInst.getSchemClass().getName().equals(ReactomeJavaConstants.GO_CellularComponent)
+						)
 						{
```

Comment:
Formatting change to reduce line-width by 1 char, but increase line-count by 2? ;)

Reply:
Yeah, okay :).  

I'm trying to keep line count to a cutoff of 120 characters so the width of the code is uniform and there's no need to horizontal scroll when working in an IDE or GitHub.  I agree, though, this adds a lot of whitespace.

Since checking the schema class name is logic that is re-used a few times, I just created a helper method to consolidate it and address the line width: https://github.com/reactome/data-release-pipeline/pull/121/commits/e72f83d95c31b984837bb1e3e7912283cbb7db5b